### PR TITLE
update list of 3rd party tool suggestions

### PR DESCRIPTION
**Pull request by deniak**
@tidoust mentioned that the default caching strategy (Cache-Control: 1 year) of staticaly can be an issue to check the latest changes of a document.
raw.githack.com provides a more detailed UI with 2 URLs (prod v. dev) but does warn about heavy usage of development URLs.
That PR updates the FAQ to mention the above.

### CODE REVIEW

File: faq.html [hunk 1]
```diff
@@ -218,14 +218,22 @@ <h3>What about serving specs using a third-party tool? <a href="#others">&#x1F51
...
+      <p>Nowadays, if you absolutely need such a service, we would probably suggest the following two options
+        &nbsp;&mdash;&nbsp;with the caveat that <em>it is just a recommendation that comes with no support nor
+          guarantee</em>, since we do <em>not</em> control these services:</p>
+        <ul>
+          <li><a href="https://www.staticaly.com/">Staticaly</a>. By default, staticaly's cache is set for
+            <em>1 year</em>, except for the files under the <code>master</code> branch. If you need the latest
+            changes, you may append the following query string <code>?env=dev</code> to the URL.
+          <li><a href="https://raw.githack.com/">raw.githack.com</a>. The UI of this service gives you URLs for
+            production and development you can use depending on your needs. <strong>Note</strong>: this service
+            is ran by a single individual and it does warn about the traffic generated by development URLs in
```
Comment (vivienlacourba) [inline faq.html]: As we discovered that **Statically** does not seem to have mush more contributors than **githack.com** I would suggest removing teh following bit: `is ran by a single individual and it`

File: faq.html [hunk 2]
```diff
@@ -218,14 +218,22 @@ <h3>What about serving specs using a third-party tool? <a href="#others">&#x1F51
...
+            production, so use it with caution.
+        </ul>
+        <p>The reason to opt for the services above is that others, like
+          <a href="https://htmlpreview.github.io/">HTMLPreview</a>, either rely on client-side JavaScript to
+          generate the page dynamically, are limited to GH repositories only, or alter the pages or the links
+          within them in some way.)</p>
```
Comment (vivienlacourba) [inline faq.html]: there is a `)` that looks extraneous.

File: faq.html (hunk 1 above)
Comment (deniak) [reply faq.html]: adressed!

File: faq.html [hunk 3]
```diff
@@ -218,14 +218,21 @@ <h3>What about serving specs using a third-party tool? <a href="#others">&#x1F51
...
-        the links within them in some way.)</p>
+      <p>Nowadays, if you absolutely need such a service, we would probably suggest the following two options
+        &nbsp;&mdash;&nbsp;with the caveat that <em>it is just a recommendation that comes with no support nor
+          guarantee</em>, since we do <em>not</em> control these services:</p>
+        <ul>
+          <li><a href="https://www.staticaly.com/">Staticaly</a>. By default, staticaly's cache is set for
+            <em>1 year</em>, except for the files under the <code>master</code> branch. If you need the latest
+            changes, you may append the following query string <code>?env=dev</code> to the URL.
+          <li><a href="https://raw.githack.com/">raw.githack.com</a>. The UI of this service gives you URLs for
+            production and development you can use depending on your needs. <strong>Note</strong>: this service
```
Comment (tidoust) [inline faq.html]: The terms "production" and "development" don't mean much by themselves, I suggest to clarify main differences with "production (1 year cache, no traffic limit) and development (changes reflected within minutes, low-traffic only)".

File: faq.html [hunk 4]
```diff
@@ -218,14 +218,21 @@ <h3>What about serving specs using a third-party tool? <a href="#others">&#x1F51
...
+            does warn about the traffic generated by development URLs in production, so use it with caution.
```
Comment (tidoust) [inline faq.html]: "development URLs in production"? That's confusing. I suggest to write:

"This service warns that excessive traffic to development URLs will be throttled."

File: faq.html (hunk 3 above)
Comment (deniak) [reply faq.html]: Text updated: https://github.com/w3c/w3c.github.io/pull/69/commits/34291d513c6505f9e8ecbf947a93c1a4759a0d71

File: faq.html (hunk 4 above)
Comment (deniak) [reply faq.html]: Text updated: https://github.com/w3c/w3c.github.io/pull/69/commits/34291d513c6505f9e8ecbf947a93c1a4759a0d71

**Pull request merged**